tests/integration/iop/test_iop_dag_pull_sftp.py

```diff
@@ -155,7 +155,7 @@ def test_dag_migrate_from_FTP(iop_empty_repo):
                 "pdf": "extracted/2022-09-24T03_01_43_content/1674-1137/1674-1137_46/1674-1137_46_10/1674-1137_46_10_103108/cpc_46_10_103108.pdf",
                 "xml": "extracted/2022-09-24T03_01_43_content/1674-1137/1674-1137_46/1674-1137_46_10/1674-1137_46_10_103108/cpc_46_10_103108.xml",
             },
-            {"xml": "extracted/aca95c.xml/aca95c.xml"},
+            {"xml": "extracted/aca95c/aca95c.xml"},
```

Comment:
this was failing too?

Reply:
it's a cosmetic change:  
before: aca95c.**xml**.zip extracted to -> "extracted/aca95c.xml/aca95c.xml"
after: aca95c.zip extracted to -> "extracted/aca95c/aca95c.xml"